GameDataParser/Parsers/MapEntityParser.cs

```diff
@@ -100,386 +67,337 @@ protected override List<MapEntityMetadata> Parse()
+                                metadata.Npcs.Add(npc);
+                                break;
                         }
-                        else if (modelData.ContainsKey("mixinMS2InteractObject"))
+                        break;
+                    case IMS2RegionSpawnBase spawnBase:
+                        switch (spawnBase)
                         {
-                            if (modelData.ContainsKey("mixinMS2InteractMesh"))
-                            {
-                                // TODO: Implement mesh packet
-                                string uuid = node.Attributes["id"].Value.ToLower();
-                                metadata.InteractMeshes.Add(new MapInteractMesh(uuid, name));
-                            }
-                            else if (modelData.ContainsKey("mixinMS2InteractActor"))
-                            {
-                                string uuid = node.Attributes["id"].Value.ToLower();
-                                metadata.InteractObjects.Add(new MapInteractObject(uuid, name, InteractObjectType.Unknown, 0));
-                            }
-                            else if (modelData.ContainsKey("mixinMS2InteractDisplay"))
-                            {
-                                // TODO: Implement Interactive Displays like 02000183, Dark Wind Wanted Board (7bb334fe41f94182a9569ab884004c32)
-                                // "mixinMS2InteractDisplay" ("ID_19100003_")
-                            }
-                            else
-                            {
-                                // TODO: Any others?
-
-                                if (name.Contains("funct_extract_"))
-                                {
-                                    string uuid = node.Attributes["id"].Value.ToLower();
-                                    metadata.InteractObjects.Add(new MapInteractObject(uuid, name, InteractObjectType.Extractor, 0));
-                                }
-                                else if (name.Contains("funct_telescope_"))
+                            case IMS2RegionBoxSpawn boxSpawn:
+                                SpawnMetadata mobSpawnDataBox =
+                                    (SpawnTagMap.ContainsKey(mapId) && SpawnTagMap[mapId].ContainsKey(boxSpawn.SpawnPointID))
+                                        ? SpawnTagMap[mapId][boxSpawn.SpawnPointID]
+                                        : null;
+
+                                int mobNpcCountBox = mobSpawnDataBox?.Population ?? 6;
+                                int mobSpawnRadiusBox = 150;
+                                // TODO: This previously relied in "NpcList" to be set. NpcList is impossible to be set on
+                                // MS2RegionSpawn, it's only set for SpawnPointNPC.
+                                List<int> mobNpcListBox = new List<int>();
+                                mobNpcListBox.Add(21000025); // Placeholder
```

Comment:
Placeholder needed?

Reply:
probably not, but it was added by the previous author. So I would leave it in until someone overhauls the system.